adal/src/main/java/com/microsoft/aad/adal/BrokerAccountServiceHandler.java

```diff
@@ -153,6 +153,9 @@ public void onError(Throwable throwable) {
         final Throwable throwable = exception.getAndSet(null);
```

Comment:
> final Throwab [](start = 7, length = 14)

Another thing that might be helpful is if we could get information on user's settings around secondary launch and background

Reply:
Unfortunately, APIs for this functionality are not available in the standard Android SDK. Huawei doesn't provide an SDK to get access to this information either: Huawei currently provides only two sets of public APIs, neither of which unblock us in this case: [Huawei ID](http://developer.huawei.com/consumer/en/service/hms/accountservice.html) and [Huawei In-App Purchases](http://developer.huawei.com/consumer/en/service/hms/payservice.html).